steps/yarn/0.0.3/step.yml

```diff
@@ -0,0 +1,45 @@
+title: Run yarn command
+summary: Runs `yarn` with the given command and args.
+description: Runs `yarn` with the given command and args.
+website: https://github.com/jonoirwinrsa/steps-yarn
+source_code_url: https://github.com/jonoirwinrsa/steps-yarn
+support_url: https://github.com/jonoirwinrsa/steps-yarn/issues
+published_at: 2016-10-20T15:24:59.660070003+02:00
+source:
+  git: https://github.com/jonoirwinrsa/steps-yarn.git
+  commit: 107ab5c1ca7b60b2089e4bba2e7288c4b18999d5
+host_os_tags:
+- osx-10.11
+project_type_tags:
+- ios
+- android
+type_tags:
+- react-native
+- npm
+- yarn
+deps:
+  brew:
+  - name: node
+  - name: yarn
+is_always_run: false
+is_skippable: true
```

Comment:
are you sure about this? is skippable: true means that if the step fails that should not mark the build as failed

this is useful for e.g. IM (Slack) steps where you don't want to mark the build as failed if the step fails, but I guess this should not be used for this step